src/components/LoanFinding/LendingCategorySection.vue

```diff
@@ -170,6 +170,8 @@ export default {
 
 				this.$refs[key][0].addToBasket(amount);
 			}
+			const { multipleAmount } = this;
```

Comment:
I meant storing the constant before calling `addToBasket`

Reply:
Oh, Any concerns triggering event before `for` statement?

Reply:
Not the tracking event, but storing `multipleAmount` in a constant above the loop before calling `addToBasket` in the loop. The `multipleAmount` return changes based on the `unreservedAmount`, which could be influenced by adding the loan to the basket.